L1Trigger/L1TMuonEndCapPhase2/src/Utils/CSCUtils.cc

```diff
@@ -0,0 +1,168 @@
+#include "L1Trigger/L1TMuonEndCapPhase2/interface/DAQ/TPrimitives.h"
+
+#include "L1Trigger/L1TMuonEndCapPhase2/interface/Utils/CSCUtils.h"
+
+namespace emtf::phase2::csc {
+
+  // Chambers
+  int next_10deg_chamber(int chamber) { return (chamber == 36) ? 1 : (chamber + 1); }
+
+  int prev_10deg_chamber(int chamber) { return (chamber == 1) ? 36 : (chamber - 1); }
+
+  int next_20deg_chamber(int chamber) { return (chamber == 18) ? 1 : (chamber + 1); }
+
+  int prev_20deg_chamber(int chamber) { return (chamber == 1) ? 18 : (chamber - 1); }
```

Comment:
I think the `36` and `18` are clear enough in context here, but you could consider adding constants for them.

Reply:
I think this is clear within the context of the function's name.